test/Graphics/Font.test.cpp

```diff
@@ -73,8 +73,8 @@ TEST_CASE("[Graphics] sf::Font", runDisplayTests())
         SECTION("Invalid data and size")
         {
             CHECK(!font.loadFromMemory(nullptr, 1));
-            const std::byte byte{0xCD};
```

Comment:
What did the compiler think was being shadowed?

Reply:
Maybe some evil global? 🤔

Reply:
> Maybe some evil global? 🤔

Yep, exactly that :(